controllers/v2/unavailability/api.py

```diff
@@ -71,3 +91,5 @@ def post(self, user_id):
 v2_api.add_resource(VolunteerUnavailabilityV2, '/v2/volunteers/',
                     '/v2/volunteers/<user_id>/unavailability')
 
+v2_api.add_resource(RemoveUnavailabilityEventV2, '/v2/volunteers/',
```

Comment:
You didn't need to add this, you could have just merged your delete function into SpecificVolunteerUnavailabilityV2